content/docs/command-reference/root.md

```diff
@@ -40,6 +40,9 @@ $ dvc root
 
 ## Example: Referencing files
 
+Simplify file references when working in a subdirectory of a <abbr>DVC
+project</abbr>.
+
 ```dvc
 $ dvc root
 
```

Comment:
```suggestion
```
Let's also remove all other empty lines in example code blocks, at least after dvc root (the actual output has no extra line).

Reply:
Did this and made the last one consistent, adding a space before the `$ tree` command begins.

Reply:
Thanks